en/08_Changelogs/5.4.0.md

```diff
@@ -126,6 +126,19 @@ See [indexes](/developer_guides/model/indexes/) to learn about indexes in Silver
 
 ## API changes
 
+### Return type changes in composer plugins
+
+We have made an exception to our [definition of public API](/project_governance/public_api/) by adding a return type to the `execute()` method in the symfony commands for `silverstripe/vendor-plugin` and `silverstripe/recipe-plugin`.
+
+The exception was made for the following reasons:
+
+- This change ensures CMS 5 can continue to be installed in scenarios where Composer is installed with Smyfony 7.
+  - This can be done manually, and is also likely to be a future default.
+- The likelihood of someone subclassing the relevant classes is very low.
+- For installations of installer with earlier versions of Symfony this won't cause any problems, since PHP allows return types to be more specific in subclasses.
```

Comment:
Optionally link to covariance on https://www.php.net/manual/en/language.oop5.variance.php#language.oop5.variance

Reply:
Done